modules/unspents/src/dimensions.ts

```diff
@@ -399,6 +399,17 @@ Dimensions.fromInput = function (input: utxolib.TxInput, params = {}) {
       case 'witnessscripthash':
         return p2wshInput;
     }
+    if (utxolib.bitgo.isParsedSignatureScriptTaproot(parsed)) {
```

Comment:
why not `case 'taproot' in L41?

Reply:
It came down to playing nice with the types. If I just add another `case` then I can't access `parsed.controlBlock` without manually casting `parsed` to `ParsedSIgnatureScriptTaproot`.

Reply:
I see